wpiutil/src/llvm/raw_ostream.cpp

```diff
@@ -845,12 +845,12 @@ raw_null_ostream::~raw_null_ostream() {
 #endif
 }
 
-void raw_null_ostream::write_impl(const char *Ptr, size_t Size) {
+void raw_null_ostream::write_impl(const char * /*Ptr*/, size_t /*Size*/) {
```

Comment:
Peter on Slack: ```it looks like llvm prevailing style is to silence variables only used in asserts with `(void)`, but silence args with `/*arg*/`.  So let's go with the latter since it's both an LLVM file and consistent with our style guide.```